src/Material/Lists.elm

```diff
@@ -0,0 +1,194 @@
+module Material.Lists
+    exposing
+        ( ul
+        , li
+        , twoLine
+        , threeLine
+        , itemPrimaryContent
+        , itemAvatar
+        , itemIcon
+        , itemSecondaryContent
+        , itemSecondaryInfo
+        , itemSecondaryAction
+        , itemTextBody
+        , itemSubTitle
+        )
+
+{-| From the [Material Design Lite documentation](http://www.getmdl.io/components/index.html#lists-section):
+
+> Lists present multiple line items vertically as a single continuous element.
+> Refer the Material Design Spec to know more about the content options.
+
+See also the
+[Material Design Specification]([https://material.google.com/components/lists.html).
+
+Refer to [this site](http://debois.github.io/elm-mdl#/lists)
+for a live demo.
+
+Example use:
+
+    ul []
+        [ li [  ] [  ]
+        , li [  ] [  ]
+        , li [  ] [  ]
+        ]
+
+# Main list elements
+@docs ul, li
+
+# List item configuration for multiple lines
+@docs twoLine, threeLine
+
+# Content configuration for one or multiple lines
+@docs itemPrimaryContent, itemAvatar, itemIcon
+
+# Content configuration for multiple lines
+@docs itemSecondaryContent, itemSecondaryInfo, itemSecondaryAction, itemTextBody, itemSubTitle
+-}
+
+import Platform.Cmd exposing (Cmd, none)
+import Html.Events as Html
+import Html exposing (Html, Attribute)
+import Parts exposing (Indexed)
+import Material.Options as Options exposing (Property, Style, cs, nop)
+
+
+{-| Main list function
+-}
+ul :
+    List (Property {} m)
```

Comment:
I tried with a m, but did not manage to compile. Not sure how to change this correctly.